index.html

```diff
@@ -9997,6 +9997,12 @@ <h5>Presentational Role Inheritance</h5>
           <li>The user interface makes the meaning and purpose of each state apparent.</li>
           <li>The user interface provides a separate method for controlling each state.</li>
         </ul>
+		<p>
+			User agents MUST calculate an implicit value for <sref>aria-level</sref> from the document structure if the author does not explicitly specify a value.
+		</p>
+		<p>
+			Authors MAY omit using nested <rref>group</rref> elements to indicate level hierarchy. When doing so, authors MUST ensure all <rref>treeitem</rref> elements in the <rref>tree</rref> have explicit values for <sref>aria-level</sref>, <sref>aria-posinset</sref>, and <sref>aria-setsize</sref>.
```

Comment:
```suggestion
			Authors MAY use nested <rref>group</rref> elements to implicitly indicate level hierarchy. Otherwise, authors MUST ensure all <rref>treeitem</rref> elements in the <rref>tree</rref> have explicit values for <sref>aria-level</sref>, <sref>aria-posinset</sref>, and <sref>aria-setsize</sref>.
```

Just a subjective editorial suggestion here. The first sentence was a little fuzzy for me on first read, so I wonder if this logic flip might be clearer but still accurate?

Reply:
Yeah, I like the rewording to positive over negative better as well 👍